packages/bot/src/stores/index.js

```diff
@@ -1,8 +1,13 @@
-import BotStore from './bot-store';
+import BotStore    from './bot-store';
+import FlyoutStore from './flyout-store';
+
+// Single instance so we can use it outside React.
+export const flyout = new FlyoutStore();
 
 export default class RootStore {
     constructor(core, ws) {
         this.bot = new BotStore(ws);
```

Comment:
If we going to implement a helper class for `active_symbol` etc. like we discuss earlier, I don't think we need to pass in `ws` to other store as a constructor anymore.